### PR TITLE
Clarification in printraw() documentation stating OS Terminal is not same as Editor Console

### PR DESCRIPTION
**Pull request by ArceusMaxis**
This is in reference to #89323 (https://github.com/godotengine/godot/issues/89323)
Target Branch: master

### CODE REVIEW

**Comment by LunarTides** [comment]
The description of the commit should be removed and the name of the commit should probably be changed to something more descriptive (I have no ideas)

**Comment by AThousandShips** [comment]
This should probably be a `[b]Note:[/b]` at the end of the description, and shouldn't mention a specific executable, as:
* Your example is Windows specific
* The exact naming depends on various things, like mono build, or if it's an official build or custom build, the custom builds are not formatted like that, they're instead `godot.windows.editor.dev.x86_64.console.exe` for example

**Comment by ArceusMaxis** [comment]
> This should probably be a `[b]Note:[/b]` at the end of the description, and shouldn't mention a specific executable, as:
> 
> * Your example is Windows specific
> * The exact naming depends on various things, like mono build, or if it's an official build or custom build, the custom builds are not formatted like that, they're instead `godot.windows.editor.dev.x86_64.console.exe` for example

Hello! i would like to learn how can i format strings to automatically refer the version of Godot the documentation is presenting to the user? What would be a good way to convey to the user the console application is necessary to see the `printraw()` method's output?

Thank you for guiding me <3

**Comment by AThousandShips** [comment]
> Hello! i would like to learn how can i format strings to automatically refer the version of Godot the documentation is presenting to the user?

That's not possible, and the documentation shouldn't be platform specific, as it can be used on different platforms, and is on the web

I'm not sure exactly how to describe it so it's clear though, look at the rest of the online documentation and see if there's some reference there

File: doc/classes/@GlobalScope.xml
Comment (AThousandShips) [inline doc/classes/@GlobalScope.xml:904]:
```suggestion
				Prints one or more arguments to strings in the best way possible to the OS terminal. Unlike [method print], no newline is automatically added at the end.
				[b]Note:[/b] The OS terminal is NOT the Editor's Console, the output can be seen using the console versions of the Godot Engine.
```

File: doc/classes/@GlobalScope.xml
Comment (AThousandShips) [inline doc/classes/@GlobalScope.xml:903]:
```suggestion
				Prints one or more arguments to strings in the best way possible to the OS terminal. Unlike [method print], no newline is automatically added at the end.
```

**Review by fire — COMMENTED** [review]
1. follow the process for squashing commits
2. The text looks fine so I'll wait for that.

**Comment by ArceusMaxis** [comment]
Hope i have squashed the commits properly (its my first time) and Thank you so much once again for guiding me throughout the contribution process, i sincerely appreciate your efforts and guidance ❤️

**Comment by AThousandShips** [comment]
Please remove the comments from different commits from the message, just keep what's important :) you can wait until it's approved though

**Comment by ArceusMaxis** [comment]
Heya! Do you have any more suggestions? Is the current commit detail okay?

**Comment by AThousandShips** [comment]
It looks good

**Review by mhilbrunner — APPROVED** [review]
I'm not super sure whether "the console versions of Godot" is clear enough or going to be confused with, well, console platforms, but I also don't have a better suggestion.

Besides that, this looks good to me.

**Comment by akien-mga** [comment]
I rewrote this in 85fb4439e34f13784be03cd85e03320e20fdbdc9 to avoid the use of "console versions", and also avoid calling the editor's Output dock a "Console", as this is not a terminology we use in the editor (and increases the confusion with the `console.exe` terminal version).

(I'd normally push that amended commit to the PR to merge it, but this PR was created without permission for maintainers to edit it, so I had to push directly to `master`.)

Superseded by 85fb4439e34f13784be03cd85e03320e20fdbdc9. Thanks for the contribution!

**Comment by ArceusMaxis** [comment]
Sorry about the incorrect settings, i wasn't aware of it, Thanks again for your efforts! 🙏❤️

**Pull request closed without merging**
